src/components/atoms/Dropdown/Dropdown.scss

```diff
@@ -87,9 +103,10 @@
       display: none;
     }
 
+    // &__Select__value-container--has-value
```

Comment:
remove?

Reply:
Keep, for now. It's one of the classes generated by the component. I should add note though.